src/main/resources/docgen/templates/car17.template

```diff
@@ -0,0 +1,39 @@
+# Anserini: Experiments on [Car17](http://trec-car.cs.unh.edu/)
+
+## Indexing
+
+Typical indexing command:
+
+```
+${index_cmds}
+```
+
+The directory `/path/to/Car17` should be the root directory of Car17 collection, i.e., `ls /path/to/Car17` should bring up a list of `.cbor` files.
+
+For additional details, see explanation of [common indexing options](common-indexing-options.md).
+
+## Retrieval
+
+Topics and qrels are stored in `src/main/resources/topics-and-qrels/`, downloaded from NIST:
+
++ `topics.car17.test200.txt`: [Topics for the test200 subset (TREC 2017 Complex Answer Retrieval Track)](http://trec-car.cs.unh.edu/datareleases/v1.5/test200-v1.5.tar.xz)
++ `qrel: qrels.car17.test200.hierarchical.txt`: [adhoc qrels (TREC 2017 Complex Answer Retrieval Track)](http://trec-car.cs.unh.edu/datareleases/v1.5/test200-v1.5.tar.xz)
```

Comment:
Why is this file called `qrels.car17.test200.hierarchical`, which is not compatible with the topics file?
I think `topics.car17.txt` and `qrels.car17.txt` is better.

Reply:
In car17, there are two test sets: `test200` and `benchmark_test`. I follow [benchmark paper](https://arxiv.org/abs/1705.04803) and use `test200` as the evaluation set in Anserini.

And for each test set, there are three kinds of `qrels` file: `article`, `toplevel`, `hierarchical`. Car17 use `hierarchical` for the final evaluation.

That's where `test200` and `hierarchical`are from. To clarify which test set and `qrels` file I use,  I add them in the file name. Is that good?

Reply:
Yea, this makes more sense to me!